content/webapp/components/AudioPlayerNew/index.tsx

```diff
@@ -168,6 +55,7 @@ export const AudioPlayer: FunctionComponent<AudioPlayerProps> = ({
   // announcement every second.
   const [startTime, setStartTime] = useState(currentTime);
   const { trackPlay, trackEnded, trackTimeUpdate } = useAVTracking('audio');
+  const { activeAudioPlayerId, setActiveAudioPlayerId } = useAppContext();
```

Comment:
To get around the (admittedly very edge) case you mentioned of having two of the same audio file on the same page, we could use the `useId` hook (`const id = useId();`) in conjunction with `setActiveAudioPlayerId`. I think?

Reply:
I like it